### ISSUE TITLE
Installation Guide and Code Walk through

### ISSUE THREAD

**Issue by Dhiraj240** [fossasia/neurolab-desktop#2]
I want to add installation guide and code overview for the project as those are not commented or properly documented.
I does need help to get the walkthrough over this project.

**Comment by arishuynhvan** [comment]
I've just managed to build this in Ubuntu using Netbeans IDE and Maven. I've typed down my steps as well. Shall I share this documentation?

**Comment by Dhiraj240** [comment]
Yes, please i have been stuck and was't able to start with this project.Please share it... i will try your steps in windows and if i get succeeded too then i will send the PR.

**Comment by arishuynhvan** [comment]
I'll leave it here for you with the links I referred to, but I will probably add it into the readme.md later. Feel free to share any feedback :)

## Set up (in Ubuntu 18.04.2)

### Dependencies

1. Java JDK & JRE
2. Maven
3. Java IDE that supports Maven

[Recommended tutorial to install in these dependencies](https://websiteforstudents.com/how-to-install-netbeans-on-ubuntu-16-04-17-10-18-04/)

### Build Neurolab

1. Clone (or fork & clone or download) Neurolab-desktop repo to the local machine.
2. [Import Neurolab-desktop as an existing Maven project in Netbeans](https://stackoverflow.com/questions/20846641/import-a-maven-project-into-netbeans)
3. Build the project in Netbeans (F11)
4. Run or explore Neurolab (F6)

**Comment by Dhiraj240** [comment]
I am unable to build the project
```
Failed to execute goal on project neurophlex: Could not resolve dependencies for project neurophlex:neurophlex:jar:0.8.0: The following artifacts could not be resolved: org.jogamp.gluegen:gluegen-rt-main:jar:2.3.2, org.jogamp.gluegen:gluegen-rt:jar:2.3.2, org.jogamp.gluegen:gluegen-rt:jar:natives-android-armv6:2.3.2, org.jogamp.jogl:jogl-all:jar:natives-linux-armv6:2.3.2: Could not transfer artifact org.jogamp.gluegen:gluegen-rt-main:jar:2.3.2 from/to central (http://repo.maven.apache.org/maven2): Connection to http://repo.maven.apache.org refused: Connection timed out: connect -> [Help 1]

To see the full stack trace of the errors, re-run Maven with the -e switch.
Re-run Maven using the -X switch to enable full debug logging.

For more information about the errors and possible solutions, please read the following articles:
[Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by arishuynhvan** [comment]
Hi, have you checked the Maven link provided with the error in your stack trace? 
http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

They provide 6 possible scenarios in which this error may occur, so probably you may want to see if your situation fell into any of those categories.

**Comment by Dhiraj240** [comment]
Its related to proxy, looking at it.Have you received any such error earlier ?

**Comment by arishuynhvan** [comment]
No, I didn't encounter it.

**Comment by Dishebh** [comment]
@arishuynhvan shall I add the above snippet to the readme if that works fine?

**Comment by arishuynhvan** [comment]
Hi Dishebh Bhayana

If it's useful, feel free too add it.
